### PR TITLE
 cleanup shortint

### PR DESCRIPTION
**Pull request by mayeul-zama**

### CODE REVIEW

**Review by tmontaigu — APPROVED** [review]

File: tfhe/src/shortint/engine/server_side/comp_op.rs [hunk 1]
```diff
@@ -36,24 +36,13 @@ impl ShortintEngine {
         ct_left: &mut Ciphertext,
         ct_right: &mut Ciphertext,
     ) -> EngineResult<Ciphertext> {
-        let mut result = ct_left.clone();
```
Comment (IceTDrinker) [inline tfhe/src/shortint/engine/server_side/comp_op.rs:39]: the above allow does not apply anymore I believe as the message extract assign is performed on ct_left

File: tfhe/src/shortint/engine/server_side/comp_op.rs [hunk 2]
```diff
@@ -90,23 +79,13 @@ impl ShortintEngine {
...
-        Ok(result)
-    }
-
-    pub(crate) fn smart_greater_or_equal_assign(
-        &mut self,
-        server_key: &ServerKey,
-        ct_left: &mut Ciphertext,
-        ct_right: &mut Ciphertext,
-    ) -> EngineResult<()> {
         if !server_key.is_functional_bivariate_pbs_possible(ct_left, ct_right) {
```
Comment (IceTDrinker) [inline tfhe/src/shortint/engine/server_side/comp_op.rs:82]: same thing throughout looks like the allow is not required anymore

**Review by IceTDrinker — COMMENTED** [review]
Small comment I believe some of the allow unused mut clippy lint are not required when removing the "comp_op_assign" ?

File: tfhe/src/shortint/engine/server_side/comp_op.rs (hunk 1 above)
Comment (mayeul-zama) [reply tfhe/src/shortint/engine/server_side/comp_op.rs:39]: Removed

File: tfhe/src/shortint/engine/server_side/comp_op.rs (hunk 2 above)
Comment (mayeul-zama) [reply tfhe/src/shortint/engine/server_side/comp_op.rs:78]: Removed

**Comment by IceTDrinker** [comment]
When the tests from the previous run pass (the once launched on Thomas' review) and the cargo builds are green good to merge as the change here just removes the allow unused mut which is not required anymore

**Comment by IceTDrinker** [comment]
integer run to check before merging : https://github.com/zama-ai/tfhe-rs/actions/runs/6146485466

**Comment by IceTDrinker** [comment]
looks good to merge Integer tests are green

**Pull request merged**